napari/layers/labels/labels.py

```diff
@@ -341,6 +355,22 @@ def mode(self, mode: Union[str, Mode]):
         self.events.mode(mode=mode)
         self.refresh()
 
+    @property
+    def overwrite(self):
+        """Defines if painting should overwrite existing labels
+
+        Defaults to true to allow overwrite to ensure consistency
+        with previous behavior, if set to false, painting would
+        only be performed on the background label.
+        """
+        return self._overwrite
+
+    @overwrite.setter
+    def overwrite(self, overwrite: bool):
+        self._overwrite = overwrite
+        self.events.overwrite(overwrite=overwrite)
+        self.refresh()
```

Comment:
I don't think the overwrite setter needs the `refresh` as changing the overwrite doesn't actually change how any of the data gets displayed - it just changes how data will get modified